### PR TITLE
save language selection to session storage

### PR DESCRIPTION
**Pull request by jsaalfeld**

### CODE REVIEW

File: openslides/core/static/js/core/base.js [hunk 1]
```diff
@@ -190,15 +190,17 @@ angular.module('OpenSlidesApp.core', [
...
     'gettextCatalog',
     'OpenSlidesPlugins',
     '$ngBootbox',
-    function (gettext, gettextCatalog, OpenSlidesPlugins, $ngBootbox) {
+    function ($sessionStorage, gettext, gettextCatalog, OpenSlidesPlugins, $ngBootbox) {
         return {
             // get all available languages
             getLanguages: function () {
-                var current = gettextCatalog.getCurrentLanguage();
+                console.log($sessionStorage);
```
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/base.js]: Please remove this line.

File: openslides/core/static/js/core/base.js [hunk 2]
```diff
@@ -311,12 +313,17 @@ angular.module('OpenSlidesApp.core', [
...
 .run([
+    '$sessionStorage',
     'gettextCatalog',
     'Languages',
-    function(gettextCatalog, Languages) {
+    function($sessionStorage, gettextCatalog, Languages) {
         // set detected browser language as default language (fallback: 'en')
-        Languages.setCurrentLanguage(Languages.getBrowserLanguage());
-
+        console.log(Languages);
```
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/base.js]: Same here.

File: openslides/core/static/js/core/base.js [hunk 3]
```diff
@@ -190,15 +190,17 @@ angular.module('OpenSlidesApp.core', [
 
 // gets all in OpenSlides available languages
 .factory('Languages', [
+    '$sessionStorage',
     'gettext',
     'gettextCatalog',
     'OpenSlidesPlugins',
     '$ngBootbox',
```
Comment (FinnStutzenstein) [inline openslides/core/static/js/core/base.js]: Can you move this dependency up?

**Comment by jsaalfeld** [comment]
Added CHANGELOG entry

**Pull request merged**